config_system/config_system/general.py

```diff
@@ -185,6 +185,15 @@ def get_options_depending_on(dependent):
     return enabled_options
 
 
+def get_warning(opt):
+    """
+    Returns the warning associated with the given config option.
+    Returns None if there isn't an associated warning.
+    """
+    config = data.get_config(opt)
+    return config['warning'] if "warning" in config else None
```

Comment:
Something else, sorry - can't we just use `config.get('warning')` here? (or `config.get('warning', None) if you want to be explicit)?

That's exactly what it's doing functionally so I guess we may as well use the right helper.

Reply:
I agree, that's slightly better. I'll have the change up in a sec...

Reply:
Thanks!